ombpdf/rawlayout.py

```diff
@@ -0,0 +1,81 @@
+from html import escape
+from decimal import Decimal
+
+
+HTML_INTRO = """\
+<!DOCTYPE html>
+<meta charset="utf-8">
+<style>
+html, body {
+    font-family: sans-serif;
+}
+
+.page {
+    border: 1px solid black;
+    position: relative;
+    display: inline-block;
+    overflow: hidden;
+}
+
+.char {
+    box-sizing: border-box;
+    background: rgba(0, 0, 0, 0.75);
+    border-left: 1px solid rgba(0, 0, 0, 0.9);
+    color: lightgray;
+    position: absolute;
+    overflow: hidden;
+    font-size: 9px;
+}
+</style>
+"""
+
+PREAMBLE = """\
+<p>
+  This page is primarily intended to show the bounding boxes of various
+  page elements. In particular, font sizes are not accurate.
+</p>
+<p>
+  Inspect this page with developer tools to obtain more details.
+</p>
+"""
+
+class PxStyleAttr:
```

Comment:
Does this need to be a class? It looks like the only use cases involve passing in `kwargs` and then treating the result like a string.

Reply:
Oh good call, I'll change it!